docs/Dockerfile

```diff
@@ -1,26 +1,28 @@
 FROM docs/base:latest
 MAINTAINER Mary Anthony <mary@docker.com> (@moxiegirl)
 
-# To get the git info for this repo
-COPY . /src
-
-COPY . /docs/content/compose/
+# Get source for the other public-sub projects; private are still not picked up
+#
+RUN svn checkout https://github.com/docker/tutorials/trunk/docs/mac /docs/content/mac \
+&& svn checkout https://github.com/docker/tutorials/trunk/docs/windows /docs/content/windows \
+&& svn checkout https://github.com/docker/tutorials/trunk/docs/linux /docs/content/linux
 
-RUN svn checkout https://github.com/docker/docker/trunk/docs /docs/content/docker
-RUN svn checkout https://github.com/docker/swarm/trunk/docs /docs/content/swarm
-RUN svn checkout https://github.com/docker/machine/trunk/docs /docs/content/machine
-RUN svn checkout https://github.com/docker/distribution/trunk/docs /docs/content/registry
-RUN svn checkout https://github.com/docker/tutorials/trunk/docs /docs/content/tutorials
-RUN svn checkout https://github.com/docker/opensource/trunk/docs /docs/content
+RUN svn checkout https://github.com/docker/docker/trunk/docs /docs/content/ 
 
+RUN svn checkout https://github.com/docker/swarm/trunk/docs /docs/content/swarm  \
+&& svn checkout https://github.com/docker/machine/trunk/docs /docs/content/machine \
+&& svn checkout https://github.com/docker/distribution/trunk/docs /docs/content/registry \
+&& svn checkout https://github.com/kitematic/kitematic/trunk/docs /docs/content/kitematic \
+&& svn checkout https://github.com/docker/opensource/trunk/docs /docs/content/opensource 
 
-# Sed to process GitHub Markdown
-# 1-2 Remove comment code from metadata block
-# 3 Change ](/word to ](/project/ in links
-# 4 Change ](word.md) to ](/project/word)
-# 5 Remove .md extension from link text
-# 6 Change ](../ to ](/project/word) 
-# 7 Change ](../../ to ](/project/ --> not implemented
-# 
-# 
-RUN /src/pre-process.sh /docs
+# Copy local Compose changes to the build
+#
+COPY . /src
+COPY . /docs/content/compose
+ 
+# Run the build ssed over the markdown to conver to Hugo crosss-references
+#
+RUN curl -sSL https://raw.githubusercontent.com/docker/docs.docker.com/master/touch-up.sh > /src/touch-up.sh \
+ && chmod +x /src/touch-up.sh \
+ && touch /docs/build.json \
+ && /src/touch-up.sh /docs
```

Comment:
Github complains about missing new line (line break) at the end of the file

In this case, you might want to add it; I don't know if Dockerfile parsing breaks if the last new line is missing. Would be a good test case btw :-)